### ISSUE TITLE
Theme switcher

### ISSUE THREAD

**Issue by banjtheman** [banjtheman/defundthepolice#22]
# What is the Task
Add a box to the sidebar to switch between dark and light theme

## Why do we want to do this
To allow different viewing experience 

## How can I get started?

1. Create a sidebar button

```
dark_theme = st.checkbox('Dark theme')
if dark_theme :
      show_dark_theme()
else:
      show_light_theme()
```

2. Create a dark theme method using code here
https://github.com/banjtheman/defundthepolice/blob/main/main_st.py#L222-L239

3. Create a light theme method

## Definition of Done
Can switch between two themes

**Comment by mpeiffer** [comment]
Hello! Can I be assigned to work on this?

**Comment by banjtheman** [comment]
closed by #56
